plugins/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(PLUGINS
 	cn0357
 	cn0508
 	cn0511
+	cn0540
```

Comment:
Please add this to the Makefile as well. I know that the plan is to move from Makefile to CMake but meanwhile this should be in both.